allennlp/service/servable/servable.py

```diff
@@ -38,16 +40,24 @@ def register(self, key: str, servable: Servable):
     def list_available(self) -> List[str]:
         return list(self.collection.keys())
 
+    # TODO: get rid of this
     @staticmethod
     def default() -> 'ServableCollection':
         import allennlp.service.servable.models.semantic_role_labeler as semantic_role_labeler
         import allennlp.service.servable.models.bidaf as bidaf
         import allennlp.service.servable.models.decomposable_attention as decomposable_attention
 
+        bidaf_params = Params(replace_none(pyhocon.ConfigFactory.parse_file(
+                'allennlp/service/servable/models/bidaf.conf')))
```

Comment:
Yeah, but as long as we have actual paths in the configuration files that point to real data, they are going to have to be changed in a test, or by a third-party user of the code.  Unless they point to some s3 location where we've stored all of the data we refer to...  But in that case we should still probably just modify it for tests.